doc/source/rllib/rllib-offline.rst

```diff
@@ -5,437 +5,2081 @@
+Each :py:class:`~ray.rllib.utils.replay_buffers.episode_replay_buffer.EpisodeReplayBuffer` instance is set with a capacity of ``1000``:
 
 .. code-block:: python
 
-    # Specify how to generate experiences:
-    #  - "sampler": Generate experiences via online (env) simulation (default).
-    #  - A local directory or file glob expression (e.g., "/tmp/*.json").
-    #  - A list of individual file paths/URIs (e.g., ["/tmp/1.json",
-    #    "s3://bucket/2.json"]).
-    #  - A dict with string keys and sampling probabilities as values (e.g.,
-    #    {"sampler": 0.4, "/tmp/*.json": 0.4, "s3://bucket/expert.json": 0.2}).
-    #  - A callable that takes an `IOContext` object as only arg and returns a
-    #    ray.rllib.offline.InputReader.
-    #  - A string key that indexes a callable with tune.registry.register_input
-    "input": "sampler",
-    # Arguments accessible from the IOContext for configuring custom input
-    "input_config": {},
-    # True, if the actions in a given offline "input" are already normalized
-    # (between -1.0 and 1.0). This is usually the case when the offline
-    # file has been generated by another RLlib algorithm (e.g. PPO or SAC),
-    # while "normalize_actions" was set to True.
-    "actions_in_input_normalized": False,
-    # Specify how to evaluate the current policy. This only has an effect when
-    # reading offline experiences ("input" is not "sampler").
-    # Available options:
-    #  - "simulation": Run the environment in the background, but use
-    #    this data for evaluation only and not for learning.
-    #  - Any subclass of OffPolicyEstimator, e.g.
-    #    ray.rllib.offline.estimators.is::ImportanceSampling or your own custom
-    #    subclass.
-    "off_policy_estimation_methods": {
-        "is": {"type": ImportanceSampling},
-        "wis": {"type": WeightedImportanceSampling}
-    },
-    # Whether to run postprocess_trajectory() on the trajectory fragments from
-    # offline inputs. Note that postprocessing will be done using the *current*
-    # policy, not the *behavior* policy, which is typically undesirable for
-    # on-policy algorithms.
-    "postprocess_inputs": False,
-    # If positive, input batches will be shuffled via a sliding window buffer
-    # of this number of batches. Use this if the input data isn't in random
-    # enough order. Input is delayed until the shuffle buffer is filled.
-    "shuffle_buffer_size": 0,
-
-The interface for a custom input reader is as follows:
-
-.. autoclass:: ray.rllib.offline.InputReader
-    :members:
-    :noindex:
-
-Example Custom Input API
-------------------------
-
-You can create a custom input reader like the following:
+    from ray.rllib.algorithms.algorithm_config import AlgorithmConfig
+
+    AlgorithmConfig()
```

Comment:
nit: Let's format these examples with the standard config format and create a `config` variable:

```
config = (
    AlgorithmConfig
    .training(
        ...
    )
    .offline_data(
        ...
    )
)
```